Validation/RecoTrack/scripts/makeTrackValidationPlots.py

```diff
@@ -54,6 +87,10 @@ def main(opts):
                         help="Save all plots separately instead of grouping them")
     parser.add_argument("--png", action="store_true",
                         help="Save plots in PNG instead of PDF")
+    parser.add_argument("--limit-tracking-algo", type=str, default=None,
+                        help="Comma separated list of tracking algos to limit to. (default: all algos; conflicts with --limit-relval)")
+    parser.add_argument("--limit-relval", action="store_true",
+                        help="Limit set of plots to those in release validation (almost). (default: all plots in the DQM files; conflicts with --limit-tracking-algo)")
```

Comment:
@lveldere The `--limit-relval` is the option you had asked earlier. Despite the name it includes a handful of plots not in the "release validation set", but those should not take much time (i.e. the most of the slowdown by additional plots is removed with this option).